app/demo-rasterizer/RDemoRunner.cc

```diff
@@ -19,6 +20,18 @@ using namespace celeritas;
 using std::cerr;
 using std::endl;
 
+namespace
+{
+//---------------------------------------------------------------------------//
+void print_time(double time_sec)
+{
+    using celeritas::color_code;
+    cerr << color_code('x') << "... " << time_sec << " s" << color_code(' ')
```

Comment:
should we use the logger here too?

Reply:
Probably yes. When I was first updating this code I was trying to make it append to the previous line (which is flushed by the logger now) but then gave up.

Reply:
OK, I've replaced these with logger. Thanks for pointing that out @pcanal !